includes/class-wc-payments-utils.php

```diff
@@ -1251,74 +1251,31 @@ public static function enqueue_style( $handle, $path = '', $deps = [], $version
 	}
 
 	/**
-	 * Returns language data: english name and native name
+	 * Converts a WP locale to a wpcom-compatible language code.
 	 *
-	 * @param string $language Language code.
+	 * @see Automattic\Jetpack\Jetpack_Mu_Wpcom\Common::get_iso_639_locale() for similar logic.
+	 * @see https://translate.wordpress.com/projects/wpcom/ for the current state of wpcom translations.
 	 *
-	 * @return array
-	 */
-	public static function get_language_data( $language ) {
```

Comment:
`get_language_data` is no longer used.

Reply:
Nice. And to confirm, it was only used to generate/render our JS settings data, which was not needed?